charts/alfresco-transform-service/templates/config-transform-router.yaml

```diff
@@ -16,7 +16,4 @@ data:
   MISC_URL: http://{{ template "alfresco-transform-service.transform-misc.name" . }}:80
   {{- end }}
   {{- toYaml .Values.transformrouter.environment | nindent 2 }}
-  {{- range $key, $val := .Values.transformrouter.livenessProbe }}
-  {{ $key }}: {{ $val | quote }}
-  {{- end }}
```

Comment:
I guess this is because router do not have that logic of transformation performance for its probes but in that we should also remove linked values

Reply:
values are already good:
```
  readinessProbe:
    path: /actuator/health
    initialDelaySeconds: 20
    periodSeconds: 60
    timeoutSeconds: 10
  livenessProbe:
    path: /actuator/health
    initialDelaySeconds: 20
    periodSeconds: 30
    timeoutSeconds: 10
```

Reply:
OK I missed the were expanded in the deplopyment